independent-publisher-connectors/OpenAI/OpenAI.swagger.json

```diff
@@ -0,0 +1,213 @@
+{
```

Comment:
@PowerRobin The name of the swagger is invalid, Can you please rename it to `"apiDefinition.swagger"`

Reply:
Renamed file to "apiDefinition.swagger"